hooks/pre-commit

```diff
@@ -1,2 +1,16 @@
 #!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

`/bin/bash` may cause errors depending on the git GUI tool.

Reply:
`#!/usr/bin/env bash` is likely to work in most cases including macOS or Linux.

Reply:
> `#!/usr/bin/env bash` is likely to work in most cases including macOS or Linux.

I applied this suggestion! Thank you!! 😃